### DISCUSSION TITLE
Does the clock module on-click action not support custom commands?

### DISCUSSION THREAD

**Discussion by angelocordero** [Alexays/Waybar#2709] [Q&A]
I want to execute a command whenever I click on the clock, although waybar just returns an error saying "unsupported action".

are custom commands really not supported or are there any special syntax? thanks

**Comment by MajorMuff** [comment]
As far as I can tell from the wiki it should work as expected. Could you post the clock module section from `config` and maybe the command you're trying to run on click?

If you have the full config and script uploaded somewhere that would of course be even better!


Edit: I just tested it and it just works. Maybe the command or script you're running is bad?
I simply added `"on-click": "alacritty"` to the clock module and it starts alacritty without any fuss.

**Reply by angelocordero** [reply]
my bad. the issue is that I placed `on-click` under actions and not directly under the module. it works now.

**Comment by angelocordero** [comment]
closing this as issue is resolved
